assets/sass/local.sass

```diff
@@ -1,5 +1,2 @@
-.govuk-main-wrapper
-  min-height: 600px
-
```

Comment:
removing this as Jas mentioned we end up adding a lot of space on a lot of pages.
It seems like a few pages were broken as missing row tags so hopefully now they are fixed this will cause no additional problems.